arches/app/templates/views/components/widgets/edtf.htm

```diff
@@ -25,80 +27,80 @@
           <div class="form-horizontal" data-bind="if: showEDTFFormats">
               <div class="edtf-style-tools-panel">
```

Comment:
this element should be an "article" with a lable like so:
`<article class="edtf-style-tools-panel" aria-label="{% trans "EDTF Formats and Examples" %}">`